index.html

```diff
@@ -45,9 +48,196 @@ <h2>Software Developer</h2>
                     <p>I can help you build a product , feature or website Look through some of my work and experience!
                         If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
                 </div>
+                <div class="hr-flex">
+                    <hr>
+                </div>
 
             </section>
+            <div>
```

Comment:
- [ optional ] It will be nice to use a `section` tag instead of a generic `div` and give it the appropriate style.